backend/bin/deal-observer-backend.js

```diff
@@ -127,12 +136,28 @@ const sparkApiSubmitDealsLoop = async (pgPool, { sparkApiBaseUrl, sparkApiToken,
  * @param {Queryable} pgPool
  */
 export const resolvePayloadCidsLoop = async (makeRpcRequest, makePayloadCidRequest, pgPool) => {
+  // Initialize contract using your RPC configuration
```

Comment:
```suggestion
  // Initialize contract
```

This comment was copied from docs, here it doesn't make sense to address with "your"